samples/otlpmetricexport/src/index.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as opentelemetry from '@opentelemetry/sdk-node';
+import {AuthClient, GoogleAuth} from 'google-auth-library';
+import {credentials} from '@grpc/grpc-js';
+import {getResourceDetectors as getResourceDetectorsFromEnv} from '@opentelemetry/auto-instrumentations-node';
+import {metrics, diag, DiagConsoleLogger} from '@opentelemetry/api';
+import {OTLPMetricExporter} from '@opentelemetry/exporter-metrics-otlp-grpc';
+import {PeriodicExportingMetricReader} from '@opentelemetry/sdk-metrics';
+
+async function getAuthenticatedClient(): Promise<AuthClient> {
+  const auth: GoogleAuth = new GoogleAuth({
+    scopes: 'https://www.googleapis.com/auth/cloud-platform',
+  });
+  return await auth.getClient();
+}
+
+diag.setLogger(
+  new DiagConsoleLogger(),
+  opentelemetry.core.getEnv().OTEL_LOG_LEVEL
+);
+
+// Express App that exports traces via gRPC with protobuf
+async function main() {
+  const authenticatedClient: AuthClient = await getAuthenticatedClient();
+
+  const sdk = new opentelemetry.NodeSDK({
+    metricReader: new PeriodicExportingMetricReader({
+      // Export metrics every 10 seconds. 5 seconds is the smallest sample period allowed by
+      // Cloud Monitoring.
+      exportIntervalMillis: 10_000,
+      exporter: new OTLPMetricExporter({
+        credentials: credentials.combineChannelCredentials(
+          credentials.createSsl(),
+          credentials.createFromGoogleCredential(authenticatedClient)
+        ),
+      }),
+    }),
+    resourceDetectors: getResourceDetectorsFromEnv(),
+  });
+  sdk.start();
+
+  // Create a meter
+  const meter = metrics.getMeterProvider().getMeter('metrics-sample');
```

Comment:
nit I think this should work
```suggestion
  const meter = metrics.getMeter('metrics-sample');
```

Reply:
done